etc/profile-m-z/min.profile

```diff
@@ -6,6 +6,10 @@ include min.local
 # Persistent global definitions
 include globals.local
 
+# Disable for now, see ___
```

Comment:
I've tested min and it breaks hard when wusc isn't included:
```
(min:9): Gtk-WARNING **: 01:57:39.194: Could not find the icon 'window-close-symbolic-ltr'. The 'hicolor' theme
was not found either, perhaps you need to install it.
You can get a copy from:
	http://icon-theme.freedesktop.org/releases

(min:9): Gtk-WARNING **: 01:57:39.195: Could not load a pixbuf from /org/gtk/libgtk/icons/16x16/status/image-missing.png.
This may indicate that pixbuf loaders or the mime database could not be found.
**
Gtk:ERROR:../gtk/gtk/gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /org/gtk/libgtk/icons/16x16/status/image-missing.png: Unrecognized image file format (gdk-pixbuf-error-quark, 3)
Bail out! Gtk:ERROR:../gtk/gtk/gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /org/gtk/libgtk/icons/16x16/status/image-missing.png: Unrecognized image file format (gdk-pixbuf-error-quark, 3)
```

Reply:
My fault chromium-common still has `whitelist /usr/share/chromium`. I need to add `ignore whitelist /usr/share/chromium` (or simply `whitelist /usr/share` if it works). However, I'm more interested in enabling wusc (and wruc). So if min works w/o a `whitelist /usr/share/some-min-dir` we can remove the ignores.